content/guidance/index.html.md

```diff
@@ -12,8 +12,8 @@ These pieces of guidance are in development by the Data Standards Authority. We
 
 | Name | Status |
 | --- | --- |
-| **[UK Government Reference Architecture for Data and APIs](referencearchitecture/)** | Draft |
-| **[Publish reference data for use across government](referencedata/)** | Draft |
 | **[API Management Guidance](apimanagement/)** | Draft |
-| **[api.gov.uk Domain Guidance](apidomain/)** | Draft |
-| **[GraphQL Guidance](graphql/)** | Draft |
+| **[UK Government Reference Architecture for Data and APIs](referencearchitecture/)** | Live |
```

Comment:
I wonder if this could be auto-generated, so it stays in line with the status of the pages without needing to do it manually. Either way, this looks good :+1:

Reply:
I think @nahteb has done an auto-generated one for the standards...

Reply:
I was just thinking the same thing @jamietanna .

Reply:
Yes with the standards page it generates the contents from what files are in the directory and gets the status from what's in the frontmatter (https://github.com/alphagov/data-standards-authority/pull/73) - can easily do the same thing with this one

Reply:
If you're free to, that'd be awesome :raised_hands: 

While lookin at it, looks like some of the frontmatter / titles need tweaking, would it be possible to do that too? :smile:

Reply:
Yeah we (I, certainly) can be a bit sloppy with keeping that frontmatter up to date too - how much of it definitely needs to be in there? The dates for example are a bit annoying to update every time.

Reply:
We've been discussing it with the Standards pages - for now, we could get Middleman to retrieve it from Git's record of the last update to the file?

Reply:
I think when we discussed this with the standards page at some point, we ruled out trying to get the date from Git because you can't differentiate between a content change and something like me or Jamie making a technical change, so it will have to be manual for now